app-frontend/src/assets/styles/sass/components/_pagination.scss

```diff
@@ -0,0 +1,96 @@
+$pagination-font-size:          14px !default;
```

Comment:
OH you meant the actual file's spacing. You can also let me worry about that too.